app/workers/expired_rent_notification_worker.rb

```diff
@@ -0,0 +1,10 @@
+class ExpiredRentNotificationWorker
+  include Sidekiq::Worker
+
+  def perform
+    expired_rents = Rent.where(rent_out: Date.current)
```

Comment:
We use Time.zone instead of Date.current because we can change zone in application.rb file, [check this](https://www.reddit.com/r/rails/comments/9crh4k/datetoday_vs_timezonenowto_date/)
```suggestion
    expired_rents = Rent.where(rent_out: Time.zone.today)
```